docs/server/source/production-nodes/setup-run-node.md

```diff
@@ -0,0 +1,128 @@
+
+Please see the [notes on NTP daemon setup](../appendices/ntp-notes.html) in the Appendices.
+
+
+## Set Up Storage for MongoDB
+
+We suggest you set up a separate storage device (partition, RAID array, or logical volume) to store the data in the MongoDB database. Here are some questions to ask:
+
+* How easy will it be to add storage in the future? Will I have to shut down my server?
+* How big can the storage get? (Remember that [RAID](https://en.wikipedia.org/wiki/RAID) can be used to make several physical drives look like one.)
+* How fast can it read & write data? How many input/output operations per second (IOPS)?
+* How does IOPS scale as more physical hard drives are added?
+* What's the latency?
+* What's the reliability? Is there replication?
+* What's in the Service Level Agreement (SLA), if applicable?
+* What's the cost?
+
+There are many options and tradeoffs.
+
+Consult the MongoDB documentation for its recommendations regarding storage hardware, software and settings, e.g. in the [MongoDB Production Notes](https://docs.mongodb.com/manual/administration/production-notes/).
+
+
+## Install and Run MongoDB
+
+* [Install MongoDB](https://docs.mongodb.com/manual/installation/)
+* [Run MongoDB (mongod)](https://docs.mongodb.com/manual/reference/program/mongod/)
+
+
+## Install BigchainDB Server
+
+### Install BigchainDB Server Dependencies
+
+Before you can install BigchainDB Server, you must [install its OS-level dependencies](../appendices/install-os-level-deps.html) and you may have to [install Python 3.4+](https://www.python.org/downloads/).
+
+### How to Install BigchainDB Server with pip
+
+BigchainDB (i.e. both the Server and the officially-supported drivers) is distributed as a Python package on PyPI so you can install it using `pip`. First, make sure you have an up-to-date Python 3.4+ version of `pip` installed:
+```text
+pip -V
+```
+
+If it says that `pip` isn't installed, or it says `pip` is associated with a Python version less than 3.4, then you must install a `pip` version associated with Python 3.4+. In the following instructions, we call it `pip3` but you may be able to use `pip` if that refers to the same thing. See [the `pip` installation instructions](https://pip.pypa.io/en/stable/installing/).
+
+On Ubuntu 16.04, we found that this works:
+```text
+sudo apt-get install python3-pip
+```
+
+That should install a Python 3 version of `pip` named `pip3`. If that didn't work, then another way to get `pip3` is to do `sudo apt-get install python3-setuptools` followed by `sudo easy_install3 pip`.
+
+You can upgrade `pip` (`pip3`) and `setuptools` to the latest versions using:
+```text
+pip3 install --upgrade pip setuptools
+pip3 -V
+```
+
+Now you can install BigchainDB Server (and officially-supported BigchainDB drivers) using:
```

Comment:
Oops, that's old text from the days when `pip install bigchaindb` did install the driver. I'll remove the text in parentheses.